src/vs/workbench/contrib/languageRuntime/browser/languageRuntimeActions.ts

```diff
@@ -409,57 +409,94 @@ const selectNewLanguageRuntime = async (
 	// Generate quick pick items for runtimes.
 	const runtimeItems: QuickPickItem[] = [];
 
-	if (activeRuntimes.length > 0) {
-		// Add a separator for active sessions.
+	// Add separator for suggested runtimes
+	const suggestedRuntimes = interpreterGroups
+		.map(group => group.primaryRuntime);
+
+	if (suggestedRuntimes.length > 0) {
 		runtimeItems.push({
 			type: 'separator',
-			label: localize('positron.languageRuntime.projectRuntimes', 'Project')
+			label: localize('positron.languageRuntime.suggestedRuntimes', 'Suggested')
 		});
-		// Add active runtimes first and foremost.
-		activeRuntimes.forEach(runtime => {
+
+		suggestedRuntimes.forEach(runtime => {
 			runtimeItems.push({
 				id: runtime.runtimeId,
 				label: runtime.runtimeName,
 				detail: runtime.runtimePath,
 				iconPath: {
 					dark: URI.parse(`data:image/svg+xml;base64, ${runtime.base64EncodedIconSvg}`),
-				},
-				picked: true
```

Comment:
One thing we could do to solve the "shows up twice when filtered" problem is add another property to `QuickPickItem` here called something like `neverShowWhenFiltered`, default false:
https://github.com/posit-dev/positron/blob/9766ef740b3ad30f3fce86d6983acddd068dce13/src/vs/platform/quickinput/common/quickInput.ts#L53
Then fix this line to hide the item when the user has a filter applied:
https://github.com/posit-dev/positron/blob/9766ef740b3ad30f3fce86d6983acddd068dce13/src/vs/platform/quickinput/browser/quickInputTree.ts#L1409

It seems like not too big a change, but I understand if it feels like scope creep. But I still like the idea of the suggested interpreters showing up twice when they're not filtered.

Reply:
oh nice! let me add that in 👀